lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -9,21 +9,22 @@
 
 <style lang="scss" scoped>
 @import '../../../../styles/settings/radiuses';
-@import '../../../../styles/settings/typography';
 @import '../../../../styles/settings/spacings';
 @import '../../../../styles/components/icons';
 @import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/typography/tokens';
+
+$badge-min-height: 20px;
 
 .badge {
+  align-items: center;
 	border-radius: $radius-s;
 	display: flex;
+  min-height: $badge-min-height;
```

Comment:
something is wrong with your code style setup

Reply:
Fixed.